src/main/clojure/clara/rules.cljc

```diff
@@ -332,6 +332,8 @@
       * :forms-per-eval - The maximum number of expressions that will be evaluated per call to eval.
         Larger batch sizes should see better performance compared to smaller batch sizes. (Only applicable to Clojure)
         Defaults to 5000, see clara.rules.compiler/forms-per-eval-default for more information.
+      * :retain-compile-ctx - Retain compilation information for better compilation failures.
+        Defaults to false, see clara.rules.compiler/retain-compile-ctx-default for more information.
```

Comment:
The reason to retain the compile-ctx is for use in durability, correct? I suggest that the docs here should mention that.  My primary concern is to make it easy for users who don't need to be concerned with the setting to rapidly skip over it (and dedicate mental bandwidth to things more relevant to their use-case).

Reply:
tried to make it clearer in the latest commit.